crates/ide_assists/src/handlers/remove_dbg.rs

```diff
@@ -24,7 +24,22 @@ pub(crate) fn remove_dbg(acc: &mut Assists, ctx: &AssistContext) -> Option<()> {
     let macro_call = ctx.find_node_at_offset::<ast::MacroCall>()?;
     let new_contents = adjusted_macro_contents(&macro_call)?;
 
-    let macro_text_range = macro_call.syntax().text_range();
+    let macro_text_range = if new_contents.is_empty() {
+        let parent = macro_call.syntax().parent()?;
+
+        let start = parent
+            .prev_sibling_or_token()
+            .and_then(|el| {
+                Some(el.into_token().and_then(ast::Whitespace::cast)?.syntax().text_range().start())
+            })
+            .unwrap_or(parent.text_range().start());
+        let end = parent.text_range().end();
+
+        TextRange::new(start, end)
```

Comment:
This straight up removes the parent if I see this right, which seems suboptimal if someone had a dbg! call as the tail expression
```rust
{
	/* some more code */
	dbg!()
}
```
if I'm not mistaken this would remove the entire block

Reply:
Fixed it, and added replacing `dbg` call with `()` in `LetExpr`